pkg/smartcontract/manifest/standard/comply.go

```diff
@@ -31,7 +31,9 @@ var checks = map[string][]*Standard{
 }
 
 // Check checks if the manifest complies with all provided standards.
-// Currently, only NEP-17 is supported.
+// If the standard parameters slice is nil, no parameters check is conducted.
```

Comment:
> the standard parameters slice

the standard method's parameters slice

Because there are also events that also have parameters.